src/modules/songs/components/SongNotes/SongNotes.tsx

```
@@ -0,0 +1,167 @@
+    return (
+      <SongDetailsItem icon="NotePencil" label="Notes">
+        <div
+          onClick={() => {
+            setIsEditingNotes(true);
+          }}
+          onKeyDown={(keyDownEvent) => {
+            if (keyDownEvent.key === "Enter" || keyDownEvent.key === " ") {
+              setIsEditingNotes(true);
+            }
+          }}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)